src/index.ts

```diff
@@ -12,33 +13,67 @@ import { looper } from './utils/looper'
 import { parseConfig } from './utils/parse-config'
 import { validateConfig } from './utils/validate-config'
 
+import {
+  closeLog,
+  openLogfile,
+  printAllLogs,
+  flushAllLogs,
+} from './utils/history'
+
 class Monika extends Command {
-  static description = 'describe the command here'
+  static description = 'Monika command line monitoring tool'
 
   static flags = {
     // add --version flag to show CLI version
     version: flags.version({ char: 'v' }),
     help: flags.help({ char: 'h' }),
-    // flag with a value (-n, --name=VALUE)
+
     config: flags.string({
       char: 'c',
       description:
-        "JSON configuration file path e.g './config.json' (default './config.json')",
-      required: true,
+        'JSON configuration filename. If none is supplied, will look for default in current path.',
       default: './config.json',
     }),
-    // flag with no value (-f, --force)
-    force: flags.boolean({ char: 'f' }),
+
+    logs: flags.boolean({
+      char: 'l',
+      description: 'print all logs',
+    }),
+
+    flush: flags.boolean({
+      description: 'flush logs',
+    }),
   }
 
   async run() {
     const { flags } = this.parse(Monika)
+    openLogfile()
+
+    if (flags.logs) {
+      printAllLogs()
+      closeLog()
+      return
+    }
+
+    if (flags.flush) {
+      const ans = await cli.prompt(
+        'Are you sure you want to flush all logs in history.db (Y/n)?'
+      )
+      if (ans === 'Y') {
```

Comment:
Do we need to change `ans` into upper case?
in case user answering with `y` (lower case)?

Reply:
Since this is a confirmation for a delete/flush ... a caps Y will ensure 2 two button press. so a definite YES want to delete logs. and prevents accidental deletion.